sdk/search/Azure.Search.Documents/CHANGELOG.md

```diff
@@ -1,14 +1,12 @@
 # Release History
 
-## 11.4.0-beta.2 (Unreleased)
+## 11.4.0-beta.2 (2021-08-10)
 
 ### Features Added
-
-### Breaking Changes
+- Support for [Azure Active Directory](https://docs.microsoft.com/azure/active-directory/authentication/) based authentication. Users can specify a [`TokenCredential`](https://docs.microsoft.com/dotnet/api/azure.core.tokencredential) when creating a `SearchClient`, `SearchIndexClient` or a `SearchIndexerClient`. The tokens use the "https://search.azure.com/.default" scope.
 
 ### Bugs Fixed
-
-### Other Changes
+- https://github.com/Azure/azure-sdk-for-net/issues/22808
```

Comment:
Do we typically just link the numbers or also provide a sentence on what the bug fix was about?

Reply:
I will add a line. For straightforward bugs, I prefer the bug title and description to do the explaining.